Example/PaymentSheet Example/PaymentSheet Example/ExampleEmbeddedElementCheckoutViewController.swift

```diff
@@ -106,8 +106,10 @@ class ExampleEmbeddedElementCheckoutViewController: UIViewController {
     func didTapCheckoutButton() {
         Task {
             // MARK: - Confirm the payment
+            view.isUserInteractionEnabled = false
             let result = await embeddedPaymentElement.confirm()
             handlePaymentResult(result)
+            view.isUserInteractionEnabled = true
```

Comment:
FYI, this is fixed here and probably shouldn't be handled by the merchant: https://github.com/stripe/stripe-ios/pull/4516

Reply:
Actually, disabling their own UI sounds right too, but ours should be disabled too